### ISSUE TITLE
Unable to run differential accessibility analysis after normalization and peaks are not annotated

### ISSUE THREAD

**Issue by leeanapeters** [GreenleafLab/ChrAccR#24]
Hi, thank you for the package!

I am experiencing an issue after loading BAM files that were generated from the nextflow atacseq pipeline. 

I am able to perform differential accessibility analysis after loading these through DsATAC.bam.

If I perform any of the normalization steps I get the error that some values in the assay are not integers. I have tried to rectify this by removing peaks with NA counts, and making sure design components are factors, but this does not help.

Additionally, my DA results are only listed by the row name and do not have any gene information.

Any help would be much appreciated. I am using R4.2.3

Thanks
